app/assets/stylesheets/govuk_publishing_components/components/_highlight-boxes.scss

```diff
@@ -26,6 +26,13 @@
   }
 }
 
+.gem-c-highlight-boxes__item-wrapper--half-width {
+  @include media(desktop) {
+    width: 50%;
+    padding: ($gutter / 6) ($gutter-half + $gutter-one-third) ($gutter-half + $gutter-one-third) 0;
```

Comment:
Is there a way we can test for this as I can't tell what the value for the padding is supposed to be.

Reply:
Do you mean adding a test? We normally just use tests to check the presence of a class - I've added a test to check `gem-c-highlight-boxes__item-wrapper--half-width` is applied when the `half_width: true` is passed to the component.

Reply:
If you mean visual check, the boxes should be 50% width and padding should be `5px 25px 25px 0`

Reply:
If you think that you'll understand what this line does in 6 months, I'm happy to approve.